definitions/procedures/restore/mongo_dump.rb

```diff
@@ -0,0 +1,34 @@
+module Procedures::Restore
+  class MongoDump < ForemanMaintain::Procedure
+    metadata do
+      description 'Restore mongo dump'
+      for_feature :pulp
+      param :backup_dir,
+            'Path to backup directory',
+            :required => true
+      preparation_steps do
+        [Checks::Mongo::DBUp.new, Checks::Mongo::ToolsInstalled.new]
+      end
+      confine do
+        feature(:mongo) && feature(:pulp)
+      end
+    end
+
+    def run
+      backup = ForemanMaintain::Utils::Backup.new(@backup_dir)
+      with_spinner('Restoring mongo dump') do |spinner|
+        feature(:service).handle_services(spinner, 'start', :only => feature(:mongo).services.keys)
+        drop_and_restore_mongo(backup, spinner)
+        feature(:service).handle_services(spinner, 'stop', :only => feature(:mongo).services.keys)
+      end
+    end
+
+    def drop_and_restore_mongo(backup, spinner)
+      spinner.update('Dropping pulp_database')
+      feature(:mongo).dropdb
+
+      spinner.update('Restoring mongo dump')
+      feature(:mongo).restore(File.join(backup.file_map[:mongo_dump][:path], 'pulp_database'))
```

Comment:
restore remote mongo database does not work
```
E, [2018-05-09 12:04:03+0200 #10368] ERROR -- : Failed executing scl enable rh-mongodb34 -- mongorestore -u pulp1 -p [FILTERED] --host mongo.host.redhat.com --port 27017 /root/backup/data/offline/mongo_dump/pulp_database, exit status 1:
 2018-05-09T12:04:03.472+0200   Failed: error connecting to db server: server returned error on SASL authentication step: Authentication failed. (ForemanMaintain::Error::ExecutionError)
```
 * hardcoded `pulp_database` name
 * missing `--db <database>` option